tests/sparse_test.py

```diff
@@ -773,6 +775,7 @@ def test_csr_spmm(self, shape, dtype, transpose):
     self.assertArraysAllClose(actual, expected)
 
 
+@unittest.skipIf(platform.system() == "Windows", "Test crashes on Windows")
```

Comment:
It makes sense that cusparse tests would fail, but do *all* BCOO tests really fail on windows?

Reply:
It seems a random selection of sparse tests fail. I've seen only these three test cases so far.

Ideally I'd diagnose the problem, but not today.

Reply:
OK, fair enough 😁